.github/workflows/main.yml

```diff
@@ -37,9 +37,9 @@ jobs:
           - Pharo32-stable
           - Pharo32-alpha
           - Pharo32-3.0
+          - Moose64-10
```

Comment:
For consistency, should this rather be `Moose64-10.0` or has the version scheme be simplified for Moose?

Reply:
We are following Pharo scheme (Pharo64-9.0 then Pharo64-10) but this is not mandatory at all. We can change it if necessary for SmalltalkCI. However, I would like not to change the url following this scheme.

Reply:
Oh right, I forgot about `Pharo64-10`. Thanks!